egs/europarl_st/SRT/lcma_srt/train.py

```
@@ -0,0 +1,2196 @@
+                continue
+
+            lang = _normalize_lang_tag(lang)
+
+            if lang is None:
+                raise KeyError("Missing supervision['language'] for an ASR sample.")
+            tags.append(lang)
+
+    if "text" in supervisions and isinstance(supervisions["text"], list):
+        assert len(tags) == len(
+            supervisions["text"]
+        ), f"The number of ASR languages ​​({len(tags)}) is inconsistent with the number of texts ({len(supervisions['text'])})."
```

Ruff (0.15.20)

[error] Invalid unescaped character zero-width-space, use "\u200B" instead

Replace with escape sequence

(PLE2515)